src/views/notifications/Latest.vue

```diff
@@ -0,0 +1,324 @@
+                                User
+                              </label>
+                              <TailSelect
+                                :class="'form-control' + (this.validation_error?.target_id != null ? ' border-theme-6' : '')"
+                                :options="{
+                                  search: true,
+                                  classNames: 'w-full'
+                                }"
+                              >
+                                <option :value="user.id" v-for="user in this.users" v-bind:key="user.id" :selected="this.single_notification.target_id === user.id">
+                                  {{ user.name }}
+                                </option>
+                              </TailSelect>
+                              <div v-if="parseInt(this.single_notification.type) !== 2" class="text-theme-6 mt-2 mb-4">
+                                Icon Type is selected!
+                              </div>
+                              <div v-if="this.validation_error?.target_id != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.type[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 mb-4">
+                            <div class="flow-root">
+                              <p class="float-left"><button class="btn btn-primary btn-md" @click="updateNotification(this.single_notification)"><SaveIcon class="w-4 h-4 mr-2"></SaveIcon>Save</button></p>
+                              <p class="float-right"><button class="btn btn-danger btn-md" @click="deleteNotification(this.single_notification.id)"><Trash2Icon class="w-4 h-4 mr-2"></Trash2Icon>Delete</button></p>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <!-- END: Notification Latest Tab -->
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from 'vue'
+import Header from './Components/Header'
+import axios from 'axios'
+import DefaultTab from './Components/DefaultTab'
+import { useToast } from 'vue-toastification'
+const toast = useToast()
+
+export default defineComponent({
+  components: {
+    DefaultTab,
+    Header
+  },
+  data() {
+    return {
+      active_tab: false,
+      validation_error: {},
+      recent_notifications: []
+    }
+  },
+  mounted() {
+    this.fetchRecentNotifications()
+  },
+  methods: {
+    fetchRecentNotifications() {
+      axios.get('notifications?recent=15')
+        .then(response => {
+          this.recent_notifications = response.data.data
+        })
+        .catch(error => {
+          console.error(error)
```

Comment:
Remove debug message if not required constantly

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/0c50ad8e187577d97a1bf16ebcefab359a94a90b